components/Astronaout.jsx

```diff
@@ -0,0 +1,18 @@
+import Lottie from 'react-lottie';
+import React from 'react';
+import animationData from '../ lotties/astronaut.json';
+
```

Comment:
Nicely done converting this back into a functional component as opposed to the class used previously.

Thanks for correcting the spelling error too! However,j ust a note that the file name is still misspelt, it is currently `astronaout.jsx` instead of `astronaut.jsx`.

Reply:
I corrected the spelling of astronaut and also fixed the space typo :)